Examples/FastMarchingSegmentation/FastMarchingSegmentation.py

```diff
@@ -21,65 +21,81 @@
+    beta = float(args[7])
+    timeThreshold = float(args[8])
+    stoppingTime = float(args[9])
 
-print(inputImage)
+    inputImage = sitk.ReadImage(inputFilename, sitk.sitkFloat32)
 
-smoothing = sitk.CurvatureAnisotropicDiffusionImageFilter()
-smoothing.SetTimeStep(0.125)
-smoothing.SetNumberOfIterations(5)
-smoothing.SetConductanceParameter(9.0)
-smoothingOutput = smoothing.Execute(inputImage)
+    print(inputImage)
 
-gradientMagnitude = sitk.GradientMagnitudeRecursiveGaussianImageFilter()
-gradientMagnitude.SetSigma(sigma)
-gradientMagnitudeOutput = gradientMagnitude.Execute(smoothingOutput)
+    smoothing = sitk.CurvatureAnisotropicDiffusionImageFilter()
+    smoothing.SetTimeStep(0.125)
+    smoothing.SetNumberOfIterations(5)
+    smoothing.SetConductanceParameter(9.0)
+    smoothingOutput = smoothing.Execute(inputImage)
 
-sigmoid = sitk.SigmoidImageFilter()
-sigmoid.SetOutputMinimum(0.0)
-sigmoid.SetOutputMaximum(1.0)
-sigmoid.SetAlpha(alpha)
-sigmoid.SetBeta(beta)
-sigmoid.DebugOn()
-sigmoidOutput = sigmoid.Execute(gradientMagnitudeOutput)
+    gradientMagnitude = sitk.GradientMagnitudeRecursiveGaussianImageFilter()
+    gradientMagnitude.SetSigma(sigma)
+    gradientMagnitudeOutput = gradientMagnitude.Execute(smoothingOutput)
 
-fastMarching = sitk.FastMarchingImageFilter()
+    sigmoid = sitk.SigmoidImageFilter()
+    sigmoid.SetOutputMinimum(0.0)
+    sigmoid.SetOutputMaximum(1.0)
+    sigmoid.SetAlpha(alpha)
+    sigmoid.SetBeta(beta)
+    # sigmoid.DebugOn()
+    sigmoidOutput = sigmoid.Execute(gradientMagnitudeOutput)
 
-seedValue = 0
-trialPoint = (seedPosition[0], seedPosition[1], seedValue)
+    fastMarching = sitk.FastMarchingImageFilter()
 
-fastMarching.AddTrialPoint(trialPoint)
+    seedValue = 0
+    trialPoint = (seedPosition[0], seedPosition[1], seedValue)
 
-fastMarching.SetStoppingValue(stoppingTime)
+    fastMarching.AddTrialPoint(trialPoint)
 
-fastMarchingOutput = fastMarching.Execute(sigmoidOutput)
+    fastMarching.SetStoppingValue(stoppingTime)
 
-thresholder = sitk.BinaryThresholdImageFilter()
-thresholder.SetLowerThreshold(0.0)
-thresholder.SetUpperThreshold(timeThreshold)
-thresholder.SetOutsideValue(0)
-thresholder.SetInsideValue(255)
+    fastMarchingOutput = fastMarching.Execute(sigmoidOutput)
 
-result = thresholder.Execute(fastMarchingOutput)
+    thresholder = sitk.BinaryThresholdImageFilter()
+    thresholder.SetLowerThreshold(0.0)
+    thresholder.SetUpperThreshold(timeThreshold)
+    thresholder.SetOutsideValue(0)
+    thresholder.SetInsideValue(255)
 
-sitk.WriteImage(result, outputFilename)
+    result = thresholder.Execute(fastMarchingOutput)
+
+    sitk.WriteImage(result, outputFilename)
+
+    image_dict = {"InputImage": inputImage,
+                  "SmoothingOutput": smoothingOutput,
+                  "GradientOutput": gradientMagnitudeOutput,
+                  "SpeedImage": sigmoidOutput,
+                  "TimeCrossingMap": fastMarchingOutput,
+                  "Segmentation": result,
```

Comment:
We don't need to display the smoothing or the gradient output images. Please remove them from the list.